server/src/auth/auth.controller.ts

```diff
@@ -1,26 +1,42 @@
-import { Controller, Post, Request, UseGuards } from '@nestjs/common';
+import { Controller, Post, Response, Request, UseGuards, Get } from '@nestjs/common';
 
 import { LoginAuthGuard } from './guards/login-auth.guard';
+import { CookieAuthGuard } from './guards/cookie-auth.guard';
 import { AuthService } from './auth.service';
+import { COOKIE_KEY } from './constants';
+
+import type { User } from '../users/entities/user.entity';
 
 @Controller('auth')
 export class AuthController {
   constructor(private authService: AuthService) {}
 
-  @Post('test')
+  @Post('login')
   @UseGuards(LoginAuthGuard)
-  async authTest() {
-    return {
-      message: 'ヽ(•‿•)ノ',
-    };
+  async login(@Request() request, @Response({ passthrough: true }) response): Promise<void> {
```

Comment:
We'll probably want to use the types for both `request` and `response`: https://docs.nestjs.com/controllers#request-object